### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by AndrejChoo** [fa1ke5/confu_vr#4]
Здравствуйте. Я наткнулся на Ваш проект в процессе поиска информации по данным платам HDMI to MIPI. У меня аналогичная плата с разрешением экрана 2560х540, которая шла в комплекте с монохромным дисплеем для фотополимерного принтера разрешением 2560х1620. Для корректного отображения картинок с такой платой необходимо сжимать изображение по вертикали или горизонтали в три раза, что очень неудобно и уменьшает разрешение. Прошивок для платы в свободном доступе нет. Хотел узнать у Вас, возможно ли сконфигурировать Ваш проект под разрешение экрана 2560х1620?
andrejchukox@yandex.ru

**Comment by fa1ke5** [comment]
вам придется найти или прочитать даталоггером MIPI последовательность для вашего дисплея. В принципе сугубо теоретически этот код можно заставить работать с любым дисплеем

**Comment by AndrejChoo** [comment]
Спасибо, что так быстро ответили.Похоже, что моя плата отличается распиновкой микроконтроллера. Так как с вашей прошивкой на usb на моей плате тишина.Так что, отложу её в долгий ящик.15.06.2023, 10:11, "fa1ke5" ***@***.***>:
вам придется найти или прочитать даталоггером MIPI последовательность для вашего дисплея. В принципе сугубо теоретически этот код можно заставить работать с любым дисплеем

**Comment by fa1ke5** [comment]
Я бы хотел предостеречь от попыток использовать опубликованный код с другими дисплеями. Плата имеет несколько ШИМ преобразователей которые подают питание на дисплей, порядок подачи питания и набор напряжений соответствует исключительно указанным мной дисплеям. Управление питанием осуществляет микроконтроллер путем управления соответствующих GPIO. Другим дисплеям этот порядок может не понравиться. Кроме того попытка залить прошивку в плату с другим типом дисплея абсолютно бессмысленна, поскольку в коде присутствует обработка статусных регистров и при неправильном дисплее инициализации не произойдет. По вашему конкретно случаю могу посоветовать попробовать понажимать кнопку режима, это одна из двух кнопок на плате, возможно там есть обработчик и произойдет смена разрешения. 
ЗЫ
Если вы слили прошивку с платы можете прислать ее мне на электронную почту klimel@yandex.ru, я не могу обещать какой то результат, но ВОЗМОЖНО получится извлечь больше информации. Кроме того мне нужна модель вашего дисплея, модель контроллера на плате управления (там не всегда stm32) или детальные фото

**Comment by AndrejChoo** [comment]
Да, на моей плате действительно установлен не stm32, a ch32f103. Его прошивку Я сдампил. И чтобы не затереть родную прошивку, впаял в плату чистый stm32f103.Но проблема ещё в том, что при попытке зашить дамп в чистый МК, будь то stm32f103 или точно такой же чистый ch32f103, плата перестает инициализироваться, причём ведёт себя совершенно одинаково и с stm32 и с ch32. Т.е. с родным МК даже без подключения дисплея, ПК плату видит как по usb, так и по hdmi, и на плате загорается зелёный светодиод, а с новыми МК непрерывно быстро  мигает синий. При этом ПК видит usb устройство, а по hdmi - ничего. Вообще не могу понять, почему так. Дамп Я считал всё 64 КБ.  16.06.2023, 12:21, "fa1ke5" ***@***.***>: Я бы хотел предостеречь от попыток использовать опубликованный код с другими дисплеями. Плата имеет несколько ШИМ преобразователей которые подают питание на дисплей, порядок подачи питания и набор напряжений соответствует исключительно указанным мной дисплеям. Управление питанием осуществляет микроконтроллер путем управления соответствующих GPIO. Другим дисплеям этот порядок может не понравиться. Кроме того попытка залить прошивку в плату с другим типом дисплея абсолютно бессмысленна, поскольку в коде присутствует обработка статусных регистров и при неправильном дисплее инициализации не произойдет. По вашему конкретно случаю могу посоветовать попробовать понажимать кнопку режима, это одна из двух кнопок на плате, возможно там есть обработчик и произойдет смена разрешения.ЗЫЕсли вы слили прошивку с платы можете прислать ее мне на электронную почту ***@***.***, я не могу обещать какой то результат, но ВОЗМОЖНО получится извлечь больше информации. Кроме того мне нужна модель вашего дисплея, модель контроллера на плате управления (там не всегда stm32) или детальные фото—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
